.github/workflows/publish.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish
+
+on:
+  push:
+    branches: [ main ]
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
+    
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
```

Comment:
What happens if you run this Action without the `contents` flag? Does that still work?

Reply:
still works, removed it